src/ticketing/types.ts

```diff
@@ -70,6 +70,7 @@ export type FareContract = {
   minimumSecurityLevel: number;
   travelRights: TravelRight[];
   qrCode: string;
+  totalAmount: string;
```

Comment:
Do we know if `totalAmount` can be undefined in some cases?

Reply:
Dont think so. I checked some of my Unknown tickets also that surprises all the time, it's an empty string.

<img width="631" alt="image" src="https://github.com/AtB-AS/mittatb-app/assets/29625161/1ce3d4c9-b70e-430a-8798-7ab91184b108">

Reply:
then it looks like I will need to guard against empty strings, thanks @Nisha1306 !

Reply:
Historically, is there a reason why we don't have these types in config-specs? @gorandalum

Reply:
The reason is that config-specs originally was specs for our configuration, and fare contracts aren't part of the configuration for the app.

Reply:
So atm, do we have a bunch of duplicate types that need to be manually updated to be kept in sync e.g. in https://github.com/AtB-AS/mittatb-app/blob/master/src/ticketing/types.ts and https://github.com/AtB-AS/webshop2/blob/main/src/tickets/types.ts?
I think we should consider to have a repository for all types in Firestore.